### PR TITLE
Setup production environment

### PR DESCRIPTION
**Pull request by dmrib**
Depends on #6

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const express = require('express');
```
Comment (HeitorBoschirolli) [inline server/server.js:1]: why use `require` instead of `import`?

File: package.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "antd": "^3.26.11",
     "dompurify": "^2.0.8",
+    "express": "^4.17.1",
```
Comment (HeitorBoschirolli) [inline package.json:8]: You used `express` in the previous commit

File: server/server.js (hunk 1 above)
Comment (dmrib) [reply server/server.js:1]: Express is run through Node.js, that doesn't support `import` syntax by default, the only way to change this is by ejecting the Webpack config, which, if I recall well, you wouldn't  really want to, haha!

File: package.json (hunk 2 above)
Comment (dmrib) [reply package.json:8]: Fixed!

**Review by HeitorBoschirolli — COMMENTED** [review]
You're not updating the version to 0.1.0 as described in the commit

**Review by HeitorBoschirolli — APPROVED** [review]
+1

**Pull request merged**